### DISCUSSION TITLE
What does  adequate collective sensemaking require? A quick sketch.

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#811] [Research]
> Collecting what was originally a twitter thread from April: https://twitter.com/rufuspollock/status/1644190679688704000

What does  adequate collective sensemaking require? A quick sketch.

![image](https://github.com/life-itself/community/assets/180658/4c48a8e7-3b2e-4b3e-9669-563eca29540e)

## Relates to the Sensemaking Gap (or "[Wisdom Gap](https://lifeitself.org/blog/2020/06/22/bridging-the-wisdom-gap)")

Adequate sensemaking is so important because of of the expanding #SensemakingGap: our world has grown more and more complex and ever further beyond our individual and collective capacities to make sense of it.

## Why is there a Sensemaking Gap and why is it growing?

In short: the world got more complex & sensemaking did not keep up

As sensemaking struggles we fall back to:

1. **Simple stories** e.g. "immigrants are taking all the jobs", or "it's raining more this summer so can't be #GlobalWarming"
2. **Overwhelm & resignation**: it's all too much, so I'll just forget about it and focus on other things

![image](https://github.com/life-itself/community/assets/180658/f95dbac4-3796-48e7-bee8-f999876d4e25)
